docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,54 +1,48 @@
 # Deploy Platformatic Applications to Fly.io
 
-## Deploying a Platformatic Runtime Application 
+## Deploying a Platformatic Watt Application 
 
-This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+This guide provides instructions on deploying a Platformatic Watt application to Fly.io. With a Watt application, you are deploying your entire application, including all services in the `web` folder.
 
+### Dockerfile for JavaScript Watt Application
 
-### Dockerfile for Runtime Application
-
-Here is an example Dockerfile for a Platformatic Runtime application:
+Here is an example Dockerfile for a Platformatic JavaScript Watt application:
 
 ```dockerfile
-FROM node:20-alpine AS builder
-
-ENV APP_HOME=/home/app/node/
-WORKDIR $APP_HOME
-
-COPY package.json package-lock.json ./
-COPY services/devotion/package.json services/devotion/package.json
-
-RUN npm ci 
-
-COPY . .
-
-RUN npx platformatic compile
-
-FROM node:20-alpine
-
-ENV APP_HOME=/home/app/node/
-WORKDIR $APP_HOME
-
-COPY package.json package-lock.json ./
-RUN npm ci --only=production
-
-COPY --from=builder $APP_HOME/dist ./dist
-
+ARG NODE_VERSION=20
+FROM node:${NODE_VERSION}-alpine 
+WORKDIR /app
+COPY package.json .
+COPY ./web/composer/package.json ./web/composer/package.json  
+COPY ./web/db/package.json ./web/db/package.json 
+COPY ./web/frontend/package.json ./web/frontend/package.json 
+RUN --mount=type=bind,source=./package.json,target=./package.json \
+    # --mount=type=bind,source=./package-lock.json,target=./package-lock.json \
+    --mount=type=cache,target=/root/.npm \
+    npm install
+RUN --mount=type=bind,source=./web/composer/package.json,target=./web/composer/package.json \
+    --mount=type=cache,target=/root/.npm \
+    npm install
+RUN --mount=type=bind,source=./web/db/package.json,target=./web/db/package.json \
+    --mount=type=cache,target=/root/.npm \
+    npm install
+RUN --mount=type=bind,source=./web/frontend/package.json,target=./web/frontend/package.json \
+    --mount=type=cache,target=/root/.npm \
+    npm install
+COPY . . 
+RUN npm run build
```

Comment:
Please use a multi-staged build, so that all the dev dependencies are not used installed in production.